ci/test/pipeline.template.yml

```diff
@@ -141,6 +141,14 @@ steps:
           queue: linux-x86_64
         coverage: skip
 
+      - id: vet-check
+        label: Cargo vet Check
+        command: bin/ci-builder run stable cargo vet check
+        timeout_in_minutes: 5
+        agents:
+          queue: linux-x86_64
+        coverage: skip
```

Comment:
We could limit this to only run when a Cargo.toml file was changed:
```
inputs:
  - Cargo.lock
  - "**/Cargo.toml"
```

Reply:
Yeah, I'm somewhat nervous about having this in the main CI pipeline as it's not deterministic. At least, the way it's currently rigged up, we're depending on audits.toml files from the `main` branches of various upstreams. One option is that we could pin those files to specific commits. Then it'd be deterministic. Another option is that we could make this a nightly only job.

Btw, @def-, depending on Cargo.lock should be sufficient. Cargo.lock is changed whenever a Cargo.toml adds or removes a dependency, which should be all that's required.

Reply:
Ah, cool, there's a lock file! That should be sufficient. I'll leave a suggestion above.

@matthewarthur I do recommend adding an `inputs` block to save on CI time:

```
inputs:
  - Cargo.lock
  - misc/cargo-vet/config.toml
```